src/components/input-time-picker/input-time-picker.tsx

```diff
@@ -706,13 +705,15 @@ export class InputTimePicker
     connectLocalized(this);
 
     let { effectiveLocale } = this;
-    if (effectiveLocale === "en-US") {
+    const normalizedLocale = effectiveLocale.toLowerCase();
```

Comment:
I think you could keep the previous code if you use `locale.getSupportedLocale`, but that's a moot point if we add a new `dayjs` (or other named) context to it.